src/MainMenu.cxx

```diff
@@ -7,6 +7,8 @@
 #include <SDL.h>
 #include "engine/UIManager.hxx"
 #include "engine/basics/Settings.hxx"
+#include "../engine/ResourcesManager.hxx"
+#include "WindowManager.hxx"
```

Comment:
Why were these added? It looks like these two lines were the only changes to `MainMenu.cxx`, so I don't know if these are necessary right now.

Reply:
window manager used later

Reply:
As in you're going to add code later that uses it? If that's what it is, then just add the includes when you put in code that needs it.